src/chart.zoom.js

```diff
@@ -201,8 +201,14 @@ function panIndexScale(scale, delta, panOptions) {
 
 function panTimeScale(scale, delta, panOptions) {
 	var options = scale.options;
-	options.time.min = rangeMinLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta));
-	options.time.max = rangeMaxLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta));
+	
+	var limitedMax = rangeMaxLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - delta));
+	var limitedMin = rangeMinLimiter(panOptions, scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) - delta));
+
+	var limitedTimeDelta = delta < 0 ? limitedMax.diff(scale.lastTick) : limitedMin.diff(scale.firstTick);
+
+	options.time.max = scale.lastTick.add(limitedTimeDelta);
+	options.time.min = scale.firstTick.add(limitedTimeDelta);
```

Comment:
How about this?
My first approach had a problem with the last movement that hits the border.

For example:
lastTick: 2017-05-01T23:00:00Z, delta (time-duration):  2 hours, rangeMax: 2017-05-02T00:00:00Z.
This would result in an effective time delta of 1 hour for max, but the min value will change 2 hours.

My intention with this code was to ensure, that the same delta is added to min and max.